mecanum-drive/robot.py

```diff
@@ -6,52 +6,38 @@
 
 import wpilib
 from wpilib.drive import MecanumDrive
+from wpilib import Joystick, PWMSparkMax, TimedRobot
```

Comment:
I believe [we standardised on avoiding `from wpilib import` over in frc-docs][frc-docs #1807] (cf. [the Zen of Python][PEP 20]). For anything that's in the root `wpilib` namespace, we should just reference `wpilib.` similarly to how the C++ examples don't do `using namespace frc;` or similar.

[frc-docs #1807]: https://github.com/wpilibsuite/frc-docs/pull/1807
[PEP 20]: https://peps.python.org/pep-0020/

Reply:
on it.